node/pkg/driver/node_utils.go

```diff
@@ -177,6 +179,23 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+
+func (n NodeUtils) DevicesAreNvme(sysDevices []string) bool {
+	args := []string{"list"}
+	out, err := n.Executer.ExecuteWithTimeout(TimeOutMultipathdCmd, nvmeCmd, args)
+	if err != nil {
+		return false
```

Comment:
what if the command fails, but they are actually nvme devices?

Reply:
that the thing, the user dont have to have nvme packages on its env.. any suggestions?

Reply:
maybe check if the command exists before?

Reply:
done